tests/test_cli.py

```
@@ -3286,5 +3308,166 @@ def mock_exists_side_effect(path):
+            "namespace": "default",
+            "image_tag": "latest",
+            "use_credentials_file": False,
+            "connection_type": "tcp",
+            "meshtastic_host": "meshtastic.local",
+            "meshtastic_port": "4403",
+            "storage_class": "standard",
+            "storage_size": "1Gi",
+        }
+
+        mock_generate.return_value = [
+            "/tmp/mmrelay-pvc.yaml",
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/mmrelay-pvc.yaml"

(S108)